### PR TITLE
[4.0] Successfully remove

### PR DESCRIPTION
**Pull request by brianteeman**
Update some strings to remove the redundant word "successfully". Don't know how these were missed?

Code Review

### CODE REVIEW

File: administrator/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -999,7 +999,7 @@ JTOOLBAR_PUBLISH="Publish"
 JTOOLBAR_PURGE_CACHE="Clear Cache"
 JTOOLBAR_REBUILD="Rebuild"
 JTOOLBAR_REBUILD_FAILED="Rebuild failed: %s"
-JTOOLBAR_REBUILD_SUCCESS="Successfully rebuilt"
+JTOOLBAR_REBUILD_SUCCESS="Toolbar rebuilt"
```
Comment (Quy) [inline administrator/language/en-GB/joomla.ini]:
```suggestion
JTOOLBAR_REBUILD_SUCCESS="Toolbar rebuilt."
```

File: api/language/en-GB/joomla.ini [hunk 2]
```diff
@@ -973,7 +973,7 @@ JTOOLBAR_PUBLISH="Publish"
 JTOOLBAR_PURGE_CACHE="Clear Cache"
 JTOOLBAR_REBUILD="Rebuild"
 JTOOLBAR_REBUILD_FAILED="Rebuild failed: %s"
-JTOOLBAR_REBUILD_SUCCESS="Successfully rebuilt"
+JTOOLBAR_REBUILD_SUCCESS="Toolbar rebuilt"
```
Comment (Quy) [inline api/language/en-GB/joomla.ini]:
```suggestion
JTOOLBAR_REBUILD_SUCCESS="Toolbar rebuilt."
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ca52bcf42c1594b81daf99affe0c3e230b229aa5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31618">issues.joomla.org/tracker/joomla-cms/31618</a>.</sub>

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on ca52bcf42c1594b81daf99affe0c3e230b229aa5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31618">issues.joomla.org/tracker/joomla-cms/31618</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31618">issues.joomla.org/tracker/joomla-cms/31618</a>.</sub>

**Comment by drmenzelit** [comment]
Thanks

**Comment by brianteeman** [comment]
thanks

**Pull request merged**